### PR TITLE
UAE/Oman: Adjust geometries, properties at all placetypes, rework localities

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1421  and https://github.com/whosonfirst-data/whosonfirst-data/issues/1472.

Fixes geometry, property, and placetype issues in the UAE, adjusting geometries at all placetypes and creates new neighbourhood records. Also adjusts geometries in Oman to match UAE boundary changes. See https://github.com/whosonfirst-data/whosonfirst-data/pull/1471#issuecomment-459462947 for commit detail.

To do:
- [x] Update hierarchies
- [x] Validate all records

### CODE REVIEW

**Comment by stepps00** [comment]
Commit detail:

392cebe: update Oman and UAE country geometries, properties, store alt geoms
  - Updates the country geometries of Oman and the UAE to better align with actual country boundaries. Updated alt geometry properties, src properties, added inception dates, and updated population values.

ffb0a3b: update region geometries, properties, store alt geoms, clean up count
  - Similar to the commit above, but work done to records with a placetype of region.

62bfd77: update county geometries, properties
  - For existing counties that stay at the county placetype, updated geometries and properties.

a46952d: deprecate leftover counties, create new county records w polygons
  - For leftover county records, deprecated them and flagged them as "not current". Adds new county records for admin2 that were not in WOF already, too.

169ec21: update locality records, new properties and geoms, superseding work
  - New properties and superseding work for localities that have more than one record and are receiving a new polygon geometry. New polygon geometries for locality polygons.
 
d682357: move leftover locality points to hoods
  - For leftover locality points within the new locality polygons, moves the locality records to neighbourhoods.

**Comment by stepps00** [comment]
**Country geometries now snapped:**
screen shot 2019-01-31 at 10 46 57 am

**Region geometries now snapped and within country bounds:**
screen shot 2019-01-31 at 10 47 16 am

**New county polygons, within region bounds:**
screen shot 2019-01-31 at 10 47 46 am

**New locality polygons, above county polygons:**
screen shot 2019-01-31 at 10 48 02 am

**New locality polygons, with new neighbourhood points:**
screen shot 2019-01-31 at 10 48 12 am

File: data/137/682/640/9/1376826409.geojson [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
...10505],[55.60077192638335,25.2003484163464],[55.52935509626076,25.24393748572646],[55.52191369016475,25.25202484786848],[55.51400719607528,25.25871694603029],[55.50625573071744,25.26375539779343],[55.4738029320597,25.28073110595585],[55.46543134997688,25.2877074240584],[55.45473432792693,25.30204763389122],[55.44884321544208,25.30677602819122],[55.43969648610403,25.30512238179256],[55.40646854019104,25.29427033101103],[55.37794315015628,25.29266836235507],[55.33245187398535,25.32708400850283],[55.33895311958053,25.32897327656002],[55.34638994644376,25.32745923370056],[55.35360400946142,25.33356133265836],[55.35936045096994,25.34816581830504],[55.36816485526218,25.37050319390816],[55.37375997852813,25.36892123399478],[55.37959841470316,25.37245858600939],[55.38640629687926,25.3807050168002],[55.39271030949824,25.37800897716573],[55.40395432567152,25.38506075685916],[55.40825915976569,25.39049022243837],[55.41771926178826,25.39472623691992],[55.44775800982433,25.38064748769148],[55.47798872310113,25.37080312816187],[55.51292199135685,25.36377513321588],[55.59746463358601,25.35961517969685],[55.60418257016954,25.36090709088961],[55.60976362519119,25.36514455787511]]],"type":"Polygon"}
```
Comment (nvkelso) [inline data/137/682/640/9/1376826409.geojson]: Please make Dubai border more rational / straight – this looks like mesoshapes interpolation. Will need to be updated in the Dubai record, and parents, children.

**Comment by stepps00** [comment]
Validated the branch locally - will QA a bit more, then merge.

**Pull request merged**